src/HotspotBle/useHotspotBle.tsx

```diff
@@ -41,6 +41,7 @@ const useHotspotBle = () => {
   const instanceRef = useRef<BleManager | null>(null)
   const [devices, setDevices] = useState<Record<string, Device>>({})
   const [device, setDevice] = useState<Device>()
+  const [onboardingClient] = useState(new OnboardingClient())
```

Comment:
Why does it need to be a part of the state?

Reply:
It doesn't, that's just one way of keeping a stable instance. That being said I shouldn't have this here at all. This needs to be moved out of the sdk, so the makers can configure their own global instance.